docs/CODEOWNERS

```diff
@@ -35,11 +35,17 @@
 /client/sysinfo/ @koute
 /client/tracing/ @koute
 
+# Documentation audit
+/primitives/runtime @paritytech/DevRel/docs-audit
```

Comment:
hmmm, not sure what is the issue: https://github.com/orgs/paritytech/teams/docs-audit

Reply:
@paritytech/docs-audit I can even mention the team like this 🤔